azure-iot-device/azure/iot/device/iothub/pipeline/pipeline_ops_iothub.py

```diff
@@ -5,7 +5,6 @@
 # --------------------------------------------------------------------------
 from azure.iot.device.common.pipeline import PipelineOperation
 
-
 # TODO: Combine SetAuthProviderOperation and SetX509AuthProviderOperation once
```

Comment:
Make sure you have pre-commit installed.  I'm pretty sure this deleted line would cause a pre-commit failure.